styles/haveibeenpwned/catppuccin.user.css

```diff
@@ -0,0 +1,411 @@
+==/UserStyle== */
+
+@-moz-document domain('haveibeenpwned.com') {
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  #catppuccin(@lookup, @accent) {
+    @rosewater: @catppuccin[@@lookup][@rosewater];
+    @flamingo: @catppuccin[@@lookup][@flamingo];
+    @pink: @catppuccin[@@lookup][@pink];
+    @mauve: @catppuccin[@@lookup][@mauve];
+    @red: @catppuccin[@@lookup][@red];
+    @maroon: @catppuccin[@@lookup][@maroon];
+    @peach: @catppuccin[@@lookup][@peach];
+    @yellow: @catppuccin[@@lookup][@yellow];
+    @green: @catppuccin[@@lookup][@green];
+    @teal: @catppuccin[@@lookup][@teal];
+    @sky: @catppuccin[@@lookup][@sky];
+    @sapphire: @catppuccin[@@lookup][@sapphire];
+    @blue: @catppuccin[@@lookup][@blue];
+    @lavender: @catppuccin[@@lookup][@lavender];
+    @text: @catppuccin[@@lookup][@text];
+    @subtext1: @catppuccin[@@lookup][@subtext1];
+    @subtext0: @catppuccin[@@lookup][@subtext0];
+    @overlay2: @catppuccin[@@lookup][@overlay2];
+    @overlay1: @catppuccin[@@lookup][@overlay1];
+    @overlay0: @catppuccin[@@lookup][@overlay0];
+    @surface2: @catppuccin[@@lookup][@surface2];
+    @surface1: @catppuccin[@@lookup][@surface1];
+    @surface0: @catppuccin[@@lookup][@surface0];
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    color-scheme: if(@lookup = latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    body {
+      background-color: @base;
+      color: @text;
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    hr {
+      border-color: @surface0;
+    }
+
+    a {
+      &, &:hover, &:focus {
+        color: @text;
+        border-bottom-color: @surface2;
+      }
+
+      &:hover {
+        background-color: fade(@accent-color, 30%);
+      }
+    }
+
+    img.pwnLogo.large.modalLogo {
+      & when (@lookup = latte) {
+        filter: brightness(0) saturate(100%) invert(27%) sepia(8%) saturate(1774%) hue-rotate(196deg) brightness(97%) contrast(81%);
+      }
+      & when (@lookup = frappe) {
+        filter: brightness(0) saturate(100%) invert(84%) sepia(13%) saturate(1084%) hue-rotate(193deg) brightness(99%) contrast(94%);
+      }
+      & when (@lookup = macchiato) {
+        filter: brightness(0) saturate(100%) invert(86%) sepia(6%) saturate(1660%) hue-rotate(195deg) brightness(99%) contrast(94%);
+      }
+      & when (@lookup = mocha) {
+        filter: brightness(0) saturate(100%) invert(81%) sepia(8%) saturate(766%) hue-rotate(191deg) brightness(106%) contrast(91%);
+      }
+    }
```

Comment:
This is fine, just make sure you are getting results with a very precision level (<0.3 on https://angel-rs.github.io/css-color-filter-generator/). I [wrote this script](https://discord.com/channels/907385605422448742/1030437391607025765/1258863184723316797) one time that you can paste in the console to keep computing results until it is within range:

```js
while (Number.parseFloat(document.querySelector('#lossDetail').firstChild.textContent.split(':')[1].trim().slice(0, -1)) > 0.3) {
  compute()
}
```

Reply:
I'm pretty sure I did, although I can re-compute to make sure I get below that threshold

Reply:
Thank you! I understand it is a pain in the ~~ass~ to do that. We are working (sorta) on adding documentation for the filter trick, and we will provide a list of the filters pre-computed in the future.